File: src/main/java/kr/nyamnyam_kr/model/domain/UpvoteModel.java
package kr.nyamnyam_kr.model.domain;

import kr.nyamnyam_kr.model.entity.PostEntity;
import lombok.Data;
import org.springframework.stereotype.Component;

@Component
@Data
public class UpvoteModel {
    private Long id;
    private Long giveId;
    private Long haveId;
    private Long postId;

}

File: src/app/model/upvote.model.ts
export interface UpvoteModel {
    id: number;
    giveId: number;
    haveId: number;
    postId: number;
}

export const initialUpvote: UpvoteModel = {
    id: 0,
    giveId: 0,
    haveId: 0,
    postId: 0
}